app/services/idv/actions/cancel_send_link_action.rb

```diff
@@ -1,13 +0,0 @@
-module Idv
-  module Actions
-    class CancelSendLinkAction < Idv::Steps::DocAuthBaseStep
-      def self.analytics_submitted_event
-        :idv_doc_auth_cancel_send_link_submitted
```

Comment:
Looks like we can remove the `idv_doc_auth_cancel_send_link_submitted` method from `AnalyticsEvents` now?

(Or should we be calling these analytics methods in the replacement steps?)